cmd/podman/unshare.go

```diff
@@ -0,0 +1,63 @@
+// +build linux
+
+package main
```

Comment:
does this need to be added to the remote client?

Reply:
Probably not - only makes sense on a local machine, it spawns a fresh terminal in the userns.

Reply:
Definitely not.

Reply:
ok, then the command should not go in main.  it needs to be added to the commands.go instead so it only shows up for the local client.